.github/workflows/optimize_svgs.yml

```diff
@@ -0,0 +1,25 @@
+name: Optimize SVGs
+
+on:
+  schedule:
+    - cron: "0 0 * * 1"
+  workflow_dispatch:
+
+permissions:
+  contents: write
+
+jobs:
+  svgs:
+    name: SVGs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Optimize SVGs
+        uses: ericcornelissen/svgo-action@v3
+        id: svgo
+      - name: Commit optimizations
+        uses: stefanzweifel/git-auto-commit-action@v4
```

Comment:
If we can configure @lawnchair-bot to commit this? CC @yasanglass

Reply:
Probably, but I can not look into it these days.